src/test/2.6.1/org/apache/hadoop/hdfs/server/namenode/GSetGenerator.java

```diff
@@ -19,40 +19,21 @@
 
 package org.apache.hadoop.hdfs.server.namenode;
 
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.ALLSSD_STORAGE_POLICY_ID;
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.COLD_STORAGE_POLICY_ID;
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.HOT_STORAGE_POLICY_ID;
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.MEMORY_STORAGE_POLICY_ID;
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.ONESSD_STORAGE_POLICY_ID;
-import static org.apache.hadoop.hdfs.protocol.HdfsConstants.WARM_STORAGE_POLICY_ID;
+import static org.apache.hadoop.hdfs.protocol.HdfsConstants.*;
```

Comment:
Oh man; google-java-format allows this? :(

Reply:
offcourse, its all human that creates complexity, google-java-format likes anything that is logical..... :)